src/style.css

```diff
@@ -0,0 +1,103 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.product-card {
+  box-sizing: border-box;
+  width: 200px;
+  background-color: #fafafa;
+  box-shadow: 0 2px 5px rgba(0, 0, 0, 0.07);
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+}
+
+.product-card__container {
+  margin: 0 14px;
+}
+
+.product-card__image {
+  width: 160px;
+  height: 134px;
+  display: block;
+  margin: 30px auto;
+}
+
+.product-card__name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+  color: #000;
+  margin: 45px 0 0;
+}
+
+.product-card__number {
+  font-size: 11px;
+  line-height: 11px;
+  margin: 6px 0 0;
+}
+
+.product-card__rating {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 16px;
+}
+
+.product-card__stars {
+  display: flex;
+}
+
+.stars {
+  background: url(images/star.svg) no-repeat center;
+  height: 15px;
+  width: 15px;
+  margin-right: 4px;
+}
+
+.stars:nth-child(-n + 4) {
```

Comment:
add modifiers, use your code from previous task, for all cases(from 1 active star to 5)